WebdriverIO.md

```diff
@@ -0,0 +1,561 @@
+
+### Checking for flakiness
+
+When submitting a pull request that adds end-to-end tests, please re-run your new tests many times to make sure they pass consistently. We will ask for screenshots in the PR thread showing 5 consecutive passes of the suite(s) where you added tests.
+
+When re-running your test, you will likely see flakes that are not related to your changes. These do not count toward the required number of passes, and they do not count against the requirement that the passes be consecutive. For example, the following would be sufficient:
+
+1. All tests pass
+2. All but one test pass. The failing test is not one you added.
+3. All tests pass
+4. All tests pass
+5. All tests pass
+6. All tests pass
+
+Please re-run your tests on CI, not locally on your machine, because flakes often appear only on CI and are not reproducible locally.
+
+### Codeowner Checks
+
+When the Automated QA Team does a codeowner review on your PR that changes the e2e tests, they will be looking to make sure that you follow all the guidance in this wiki page. In the checklist below, we list some of the most common problems we see. To get your PR merged faster, please check that your PR satisfies each item:
+
+* [ ] All constants should be in all-caps. (This isn't really an e2e test issue, but we see it a lot.)
+* [ ] All element selectors, e.g. `$('.webdriverio-test-my-element')`, need to be at the top of the file. There are a few exceptions:
+    * Keeping selectors with the code that uses them is okay in some utility files where the utilities do not generally share selectors.
+    * When you are chaining selectors, only the root selector (first in the chain) needs to be at the top of the file.
+* [ ] Any time you create something in Oppia that needs a globally unique name to be identified by the tests (e.g. explorations, topics, skills, and users), make sure to follow the naming guidance in the [Independence](https://github.com/oppia/oppia/wiki/WebdriverIO#independence) section above.
+* [ ] Each `it` block in the test must be able to be run independently of the others. Imagine running any number of your `it` blocks in any order. Regardless of which you choose and in what order you run them, they should work correctly.
+* [ ] Before you interact with _any_ element on the page, you must wait for it to be present, visible, or clickable (depending on how you interact with it) using the [`waitFor`](https://github.com/oppia/oppia/blob/develop/core/tests/webdriverio_utils/waitFor.js) functions. The [`actions`](https://github.com/oppia/oppia/blob/develop/core/tests/webdriverio_utils/action.js) functions handle this for you.
```

Comment:
The waitFor and action files don't exist either. Ditto elsewhere

Reply:
Removed the links for now, I will add the links when the files will be available.